tests/apps/service_invocation_external/service_invocation_external/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
Could we rename this dir? Looks duplicate.

Reply:
done